corehq/apps/translations/app_translations/upload_module.py

```diff
@@ -115,12 +115,10 @@ def _update_case_search_labels(self, rows):
         properties = self.module.search_config.properties
         displays = [row for row in self.condensed_rows if row['list_or_detail'] == 'case_search_display']
         hints = [row for row in self.condensed_rows if row['list_or_detail'] == 'case_search_hint']
-        if len(displays) != len(hints) or len(displays) != len(properties):
-
+        if len(displays) != len(properties):
             message = _(
-                'Expected {expected_count} case_search_display and case_search_hint '
-                'properties in  menu {index}, found {actual_label_count} for case_search_display and '
-                '{actual_hint_count} for case_search_hint'
```

Comment:
`actual_hint_count` should be deleted from the format argument

Reply:
true thanks for catching that